src/locale/locales/ko/messages.po

```diff
@@ -46,10 +58,18 @@ msgstr "{0, plural, other {#}}분"
 msgid "{0, plural, one {# month} other {# months}}"
 msgstr "{0, plural, other {#}}개월"
 
+#: src/screens/Post/PostQuotes.tsx:41
+msgid "{0, plural, one {# quote} other {# quotes}}"
+msgstr "{0, plural, other {#}} 인용"
+
 #: src/view/com/util/post-ctrls/RepostButton.tsx:69
 msgid "{0, plural, one {# repost} other {# reposts}}"
 msgstr "{0, plural, other {#}}개"
 
+#: src/screens/Post/PostRepostedBy.tsx:41
+msgid "{0, plural, one {# reposts} other {# reposts}}"
```

Comment:
FYI #7064 just got merged to fix a typo here

Reply:
Thank you!